storage/innobase/xtrabackup/src/backup_mysql.h

```diff
@@ -58,6 +58,10 @@ struct log_status_t {
   lsn_t lsn_checkpoint;
   std::vector<replication_channel_status_t> channels;
   std::vector<rocksdb_wal_t> rocksdb_wal_files;
+  void clear() {
+    channels.clear();
+    rocksdb_wal_files.clear();
```

Comment:
is it OK to clear all fields? :)

Reply:
yes, they will be populated once again by `log_status_storage_engines_parse` and `log_status_replication_parse` using the new information queried by the log_status table. The main problem with those two is that they are vectors that receive the new data via append mode (push_back). The other info, like for InnoDB we just replace by a normal assignment (var = new_value).

Reply:
Ok then, lets clear everything for clarity purpose

Reply:
make sense. Done